modules/sdk-coin-atom/src/lib/iface.ts

```diff
@@ -24,15 +37,15 @@ export interface FeeData {
  */
 export interface TxData extends AtomTransaction {
   id?: string;
+  type?: TransactionType;
```

Comment:
is there a reason id and type are both optional? they both seem important

Reply:
both are optional in the baseTransaction class